### PR TITLE
Added user sign in event

### PR DESCRIPTION
**Pull request by j-maynard**
Also updated dashboard to show user id
Also added cognito_pool_id to the secrets

Co-Authored-By: Rosa Fox <rosa.fox@digital.cabinet-office.gov.uk>

### CODE REVIEW

File: spec/models/user_sign_in_event_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+RSpec.describe UserSignInEvent, type: :model do
+  include Utilities::Configuration::Settings
+
+  user_id = SecureRandom.uuid
+  let(:user_id) { SecureRandom.uuid }
+   let(:event) { UserSignInEvent.create(user_id: user_id) }
+
+  context '#create' do
+    it 'creates a valid event which contains hard-coded data' do
```
Comment (brenetic) [inline spec/models/user_sign_in_event_spec.rb]: ~hard-coded~ => user

File: spec/models/user_sign_in_event_spec.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      expect(event.data).to include(
+        'user_id'
+      )
+      expect(event.data['user_id']).to eq(user_id)
```
Comment (brenetic) [inline spec/models/user_sign_in_event_spec.rb]: Do we need to check it twice?

Comment (j-maynard) [reply spec/models/user_sign_in_event_spec.rb]: yes... They check different pits of the data

File: spec/models/user_sign_in_event_spec.rb (hunk 1 above)
Comment (j-maynard) [reply spec/models/user_sign_in_event_spec.rb]: Fixed

**Comment by brenetic** [comment]
Can we remove the `.env.swp` please

File: config/secrets.yml [hunk 3]
```diff
@@ -3,6 +3,7 @@ default: &default
     cognito_aws_access_key_id: <%= ENV["COGNITO_AWS_ACCESS_KEY_ID"] %>
     cognito_aws_secret_access_key: <%= ENV["COGNITO_AWS_SECRET_ACCESS_KEY"] %>
     cognito_client_id: <%= ENV["AWS_COGNITO_CLIENT_ID"] %>
+    cognito_userpool_id: <%= ENV["COGNITO_USERPOOL_ID"] %>
```
Comment (jakubmiarka) [inline config/secrets.yml]: do we need the userpool_id for getting the user ID?

Comment (j-maynard) [reply config/secrets.yml]: No... Sorry this was added while I was investigating getting usernames in the event we need to see more than a GUID... If you do need to see more than a GUID then we'll need this for sure as it allows querying Cognito for user and group information.

Can remove for now but we will need it soon.

**Pull request merged**